### ISSUE TITLE
project meetings in January

### ISSUE THREAD

**Issue by TanyaNikitina** [KatyaAplonova/ReportedDiscourseProject#51]
The following look like good dates for our project meetings in January:
- Thursday Jan 9, at 9am,
- Tuesday Jan 28, at 9am.
Let me know if you have objections. We could also start a little later if anyone has restrictions/preferences.

**Comment by IndianaTones** [comment]
To those of you who will be there at the meeting this Thursday, would it be ok if we discuss the following two points?

- Difficult, if not impossible, queries in ELAN and potential solutions
- Workshops and conferences in 2020

Thanks and looking forward to seeing you all!

**Comment by TanyaNikitina** [comment]
Yes, that sounds like a good plan! Katya, Lacina -- do you also plan to present something?

**Comment by LacinaSilue** [comment]
Hi all,

I plan to present something on that day. Thanks.


Lacina




Le lun. 6 janv. 2020 à 20:38, TanyaNikitina <notifications@github.com> a
écrit :

**Comment by KatyaAplonova** [comment]
Hi! Sorry for taking so long to reply. I’m making a draft of a questionnaire for Bashkir (it is in Russian, but if everybody is interested, I will translate it and show it during the meeting). Moreover, I had a closer look to Chuvash texts and I can make a short overview of the ways of expression of reported discourse in comparison with Bashkir data (I’m also thinking to write an abstract about that). And finally, perhaps, it would be useful to show my Ginyanga data, as I’m going to a field trip in the end of January.  

Happy to see you all soon!

**Comment by TanyaNikitina** [comment]
Perfect! Does anybody mind starting at 10 instead of 9?
